src/azul/lambda_layer.py

```diff
@@ -82,6 +82,8 @@ def _build_package(self, input_zip, output_zip):
 
     @cached_property
     def object_key(self):
-        path = Path(config.project_root) / 'requirements.txt'
-        sha1 = file_sha1(path)
-        return f'{config.lambda_layer_key}/{sha1}.zip'
+        requirements = Path(config.project_root) / 'requirements.txt'
+        requirements_trans = Path(config.project_root) / 'requirements.trans.txt'
+        sha = file_sha1(requirements)
+        sha_trans = file_sha1(requirements_trans)
+        return f'{config.lambda_layer_key}/{sha}.{sha_trans}.zip'
```

Comment:
```suggestion
        sha = '.'.join(file_sha1(x) for x in (requirements, requirements_trans))
        return f'{config.lambda_layer_key}/{sha}.zip'
```
I wonder if consolidating the `sha1` operation to a single line is appeals to you? Let me know what you think.